configure.ac

```diff
@@ -891,6 +891,7 @@ for option in $enable_fast ; do
     case "$option" in
         O*)
         enable_fast_opts=$option
+        enable_fast_sse_instr=yes
```

Comment:
It doesn't make sense to set the default here. Move it outside.

Reply:
WIll fix.